jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/CDataXMLStreamWriter.java

```diff
@@ -0,0 +1,58 @@
+/* 
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.process.workitem.webservice;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import org.apache.cxf.staxutils.DelegatingXMLStreamWriter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CDataXMLStreamWriter extends DelegatingXMLStreamWriter {
+
+    private static Logger logger = LoggerFactory.getLogger(CDataXMLStreamWriter.class);
+    private List<String> cdataElements = new ArrayList<>();
```

Comment:
You do not need to do this assigment (which is always performed by constructor)
In order to avoid this in the future you can declare this variable final so compiler ensures that

Reply:
right, done